front/src/applications/stdcm/components/StdcmForm/StdcmOpSchedule.tsx

```diff
@@ -52,6 +53,7 @@ const StdcmOpSchedule = ({
   opScheduleTimeType,
   opToleranceValues,
   opId,
+  isOrigin,
```

Comment:
This way, you don't have to pass `isOrigin={false}` in `StdcmDestination`

```suggestion
  isOrigin = false,
```

Reply:
Good catch ! Done

Reply:
Don't forget to remove the props in `StdcmDestination` :)